tests/dGameTests/dGameMessagesTests/GameMessageTests.cpp

```diff
@@ -1,15 +1,50 @@
 #include "GameMessages.h"
 #include "GameDependencies.h"
+#include "AMFDeserialize.h"
+
+#include "AddActionMessage.h"
+#include "AddStripMessage.h"
+#include "AddMessage.h"
+#include "MigrateActionsMessage.h"
+#include "MoveToInventoryMessage.h"
+#include "MergeStripsMessage.h"
+#include "RearrangeStripMessage.h"
+#include "RemoveActionsMessage.h"
+#include "RemoveStripMessage.h"
+#include "RenameMessage.h"
+#include "SplitStripMessage.h"
+#include "UpdateActionMessage.h"
+#include "UpdateStripUiMessage.h"
+
 #include <gtest/gtest.h>
+#include <fstream>
 
-class GameMessageTests : public GameDependenciesTest {
-	protected:
-		void SetUp() override {
-			SetUpDependencies();
-		}
-		void TearDown() override {
-			TearDownDependencies();
+class GameMessageTests: public GameDependenciesTest {
+protected:
+	void SetUp() override {
+		SetUpDependencies();
+	}
+	void TearDown() override {
+		TearDownDependencies();
+	}
+
+	std::string ReadFromFile(std::string filename) {
+		std::ifstream file(filename, std::ios::binary);
+		std::string readFile;
+		while (file.good()) {
+			char readCharacter = file.get();
+			readFile.push_back(readCharacter);
 		}
```

Comment:
There are much better ways to turn an ifstream into a string, I'm sure we have at least 5 of them in DLU already

Reply:
there probably are!  I just don't know where to look since they arent in general utils :P